doc/usecases/snippet_sharing/UC_edit_snippet.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Edit configuration snippet
+
+## Summary
+
+Title: Edit configuration snippet
+Scope: Entry Management
+Level: User Goal
+Actors: Authenticated user (snippet owner), Moderator
+Brief: Users can edit their own database entries. Moderators can edit all configuration snippets in the database, also the ones of other users.

```

Comment:
Most of the oAuth process would be frontend stuff, which is not the problem (there is an [awesome angular library](https://github.com/sahat/satellizer) for that), but it also requires the server to validate the access token by doing calls to GitHub (obviously). I'm going to do other stuff first before investing time into that.

Your link for the issue creation is an API call that creates an issue directly; it does not open the GitHub form with pre-filled fields for you. Implementing a second GitHub frontend doesn't seem to be the best idea...